### ISSUE TITLE
err log files not created on MN3

### ISSUE THREAD

**Issue by kinow** [BSC-ES/autosubmit#206]
In GitLab by @dmanubens on Nov 2, 2016, 15:51

When running AS 3.7.4  `.err` files are not created.
The header is malformed and extenison `.out` is used for err and out:
 ```
#BSUB -J a00w_19900101_fc0_1_SIM
#BSUB -oo /gpfs/scratch/pr1eaq00/pr1eaq03/a00w/LOG_a00w/a00w_19900101_fc0_1_SIM.20161102154333.out
#BSUB -eo /gpfs/scratch/pr1eaq00/pr1eaq03/a00w/LOG_a00w/a00w_19900101_fc0_1_SIM.20161102154333.out
#BSUB -W 01:00
#BSUB -n 130
```

**Comment by kinow** [comment]
In GitLab by @ jlope2 on Nov 2, 2016, 16:13

Mentioned in commit a8b0e396a4e76d71e3446b8ad2ca8a39f9ce16db

**Comment by kinow** [comment]
In GitLab by @ jlope2 on Nov 2, 2016, 16:14

Status changed to closed by commit a8b0e396a4e76d71e3446b8ad2ca8a39f9ce16db
